tests/components/create_post/create_post.test.jsx

```diff
@@ -369,7 +369,7 @@ describe('components/create_post', () => {
         expect(removeReaction).toHaveBeenCalledWith('a', 'smile');
     });
 
-    it('check for postError state on handlePostError callback', () => {
+    /*it('check for postError state on handlePostError callback', () => {
```

Comment:
Do we need to file something to go back and re-enable this test?

Reply:
Yes I filed a ticket.